docs/manual/02_concepts/08_avatars/01_animations.md

```diff
@@ -1,5 +1,83 @@
+
+### Export Setup
+When you are ready to export your animation file as GLB, the export will require some minor settings changes to ensure the root motion and animation tracks are exported correctly. 
+
+First, ensure the armature you’re exporting has all transforms applied. An unapplied rotation value will cause root motion axis misalignments. To fix this, press CTRL+A, and select `Rotation` to apply the current rotation.
+
+In the GLB export window, you must set the following values.
+
+If you are using NLA tracks, set the `Animation mode` dropdown to `NLA Tracks`, otherwise set it to `Actions`.
+
+In `Rest & Ranges`, select `Limit to Playback Range`. This prevents further root motion axis issues.
+
+![Export Presets](./images/animation-export-presets.jpg)
+
+### Creating Animation Packs
+When multiple Mixamo animation files are present with a single track, it is best practice to bundle them into one glb with multiple animation clips. You may also want to import custom animations via the Mixamo rig for retargeting onto avatars. This is the recommended workflow using Blender.
+
+#### 1. Import every Mixamo animation into a single Blender scene
+- Delete meshes bound to the rigs, if any.
+- Choose one of the imported animated rigs to be your main rig, which will hold the animation clips of every other imported rig. 
+- Optionally you may rename the main rig to keep track of it.
+![Export Presets](./images/animation-pack1.jpg)
+
+#### 2. Start packing the animations into the main rig
+- Open the Animation workspace tab
+![Export Presets](./images/animation-pack2.jpg)
+- In the top right, in the window adjacent to the viewport, open the editor type window, and select `Nonlinear Animation`
+![Export Presets](./images/animation-pack3.jpg)
+- Double click the name of every animation action to rename it to describe itself, if necessary.
+![Export Presets](./images/animation-pack4.jpg)
+- For the main rig in the NLA editor, click the `Push Down Action` button to create an NLA strip out of the action.
+![Export Presets](./images/animation-pack5.jpg)
+- This process will be repeated with the animation action of every other animated rig you’ve imported. This will require adding their animation actions to the main rig. After pushing down the top action, you will see a new `<No Action>` entry appear.
+![Export Presets](./images/animation-pack6.jpg)
+- Click the `<No Action>` entry, then select the small icon next to the New button to browse actions. Select one that isn’t assigned to the main rig yet.
+- This will add it to the top action. Push it down and repeat the process until every action has an NLA strip under the main rig.
+Now the animations have all been bundled into one rig. Now prep the rig and export it.
+- Delete every rig except for the main rig. 
+- Select the main rig, press CTRL+A to open the Apply Transform menu and select `Rotation`. This will ensure the armature object’s rotation is zeroed out to prevent root motion axis misalignments.
```

Comment:
```suggestion
- Select the main rig, press **CTRL+A** to open the Apply Transform menu and select **Rotation**. This will ensure the armature object’s rotation is zeroed out to prevent root motion axis misalignments.
```

Fixed bold formatting and keyboard inputs.